app/index.js

```diff
@@ -19,16 +19,16 @@ module.exports = yeoman.generators.Base.extend({
 			type: 'input',
 			name: 'componentName',
 			message: 'How do you want to name your class?',
-			default: 'Select Input',
+			default: 'selectInput',
```

Comment:
This should start with a capital letter actually, since it's meant to be the class name.